pkgs/top-level/all-packages.nix

```diff
@@ -1578,9 +1578,8 @@ with pkgs;
   aefs = callPackage ../tools/filesystems/aefs { };
 
   aegisub = callPackage ../by-name/ae/aegisub/package.nix ({
-    boost = boost179;
-    luajit = luajit.override { enable52Compat = true; };
-    wxGTK = wxGTK32;
+    ffmpeg = ffmpeg;
+    wxGTK = wxGTK31;
```

Comment:
Is there any workaround with wxGTK32? wxGTK31 is deprecated since 2022.

Reply:
I prefer waiting the upstream to update this.

Reply:
> Is there any workaround with wxGTK32? wxGTK31 is deprecated since 2022.

The build log shows it's no matter with its version because it reports the same error in wx32 and wx31:) Because the error is related with wx31's source so I did it temporarily, When it could be built well in Mac OS I will update to wx32.

Now the build process is trapped in the last step: link, here is [build log in Mac OS action](https://github.com/Hobr/hobr_nur/actions/runs/8864633282/job/24340022031), the error report points out the Aegisub src's src_osx_scintilla_ime.mm which is related to [wxWidgets 3.1.7's window.mm](https://github.com/wxWidgets/wxWidgets/blob/v3.1.7/src/osx/cocoa/window.mm), hope ofborg could bring a good message.